Aftonbladet.js

```diff
@@ -0,0 +1,209 @@
+{
+	"translatorID": "235c5c47-0c04-4d77-9afe-e22265a017a9",
+	"label": "Aftonbladet",
+	"creator": "Jonatan Svensson Glad",
+	"target": "^https?://ww*\.aftonbladet\.se",
```

Comment:
Copy and paste the first line into scaffold OR copy and paste the second line if you are using any text editor directly.